configs/manta.yml

```diff
@@ -0,0 +1,20 @@
+endpoint: wss://ws.manta.systems
+mock-signature-host: true
+block: ${env.MANTA_BLOCK_NUMBER}
+db: ./db.sqlite
+
+import-storage:
+  Sudo:
+   Key: dfbMFtQXqiKHH4d2y4CFcVQCpopqKcaEiYv2DCUpVCphoz8kB
+  System:
+    Account:
+      -
+        -
+          - dfbMFtQXqiKHH4d2y4CFcVQCpopqKcaEiYv2DCUpVCphoz8kB
+        - data:
+            free: 1000000000000000
+  ParachainStaking:
+    # NOTE: MANTA_BLOCK_NUMBER should set to 2170792 in .env file.
```

Comment:
This config is not useful  if users have to manually change it. Configs in this repo are meant to start the fork without any aditional work, just `chopsticks -c acala` and it use the predefined config. This config should work for any block otherwise I think it's best to guide your users how to do a proper setup

Reply:
see if you can find a way to cheat nimbus

Reply:
if this config only works for a block number, then we should just hardcoded the block number here

Reply:
I tried but it will build a few blocks then fail when nimbus rotates author